dbt/include/athena/macros/adapters/relation.sql

```diff
@@ -1,5 +1,5 @@
 {% macro drop_relation(relation) -%}
-  {% if config.get('table_type') != 'iceberg' and config.get('incremental_strategy') != 'append' %}
```

Comment:
@Jrmyy looking at gitblame seems that this was added by you, any specific reasons why??

Reply:
I did not add this, it was here before : https://github.com/dbt-athena/dbt-athena/pull/79/files#diff-72d05504f9995a78fcddb65052d2c04565d0708899e6fc4ab83cbc8cbf7695d0L2

Reply:
Mmm I checked in a rush then,sorry about that.
Any concern about removing it?